dev/tools/merge-pr.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -e
+
+# This script depends (at least) on git and jq.
+# It should be used like this: dev/tools/merge-pr.sh /PR number/
+
+#TODO: check arguments and show usage if relevant
+
+PR=$1
+
+git fetch github
+
+BASE_BRANCH=`curl https://api.github.com/repos/coq/coq/pulls/$PR 2>/dev/null | jq -r '.base.label'`
```

Comment:
Here, another solution would be to use `git fetch ${remote} refs/pull/${PR}/head`.